naucse/routes.py

```diff
@@ -122,16 +122,23 @@ def prv_nxt_teller(run, lesson):
         for material in session.materials
         if material.lesson
     ]
+
+    prv_link, prv_title, nxt_link, nxt_title = None, None, None, None
+
     for prev, current, next in zip([None] + lessons,
                                    lessons,
                                    lessons[1:] + [None]):
         if current.slug == lesson.slug:
             if prev:
                 prev = prev.index_page
+                prv_link = str(prev)
+                prv_title = prev.title
             if next:
                 next = next.index_page
-            return prev, next
-    return None, None
+                nxt_link = str(next)
+                nxt_title = next.title
+    
+    return prv_link, prv_title, nxt_link, nxt_title
```

Comment:
It would be nice to have a class to group (link, title). Something like a tooltip might be added in the future.
Could you use a `namedtuple` or `SimpleNamespace` instead of two variables for each link?